examples/rbpf_plugin.rs

```diff
@@ -16,19 +16,48 @@ _unwind(a: u64, _b: u64, _c: u64, _d: u64, _e: u64) -> u64
  * It reads the program from stdin.
  */
 fn main() {
-    let args: Vec<String> = std::env::args().collect();
-
+    let mut args: Vec<String> = std::env::args().collect();
+    let mut jit : bool = false;
     let mut program_text = String::new();
     let mut memory_text = String::new();
 
-    if args.len() == 2 {
-        memory_text = args[1].clone();
+    args.remove(0);
+    
+    // Memory is always the first argument.
+    if args.len() > 0 {
+        memory_text = args[0].clone();
         // Strip whitespace
         memory_text.retain(|c| !c.is_whitespace());
+        args.remove(0);
+    }
+
+    // Process the rest of the arguments.
+    while args.len() > 0 {
+        // Show help.
+        if args[0] == "--help" {
```

Comment:
```suggestion
match args[0] {
    "--help" => { ... },
    "--jit" => { ... },
    "--program" if args.len() > 1 => { ... },
    _ => { unknown arg },
}
```

Reply:
What is the canonical way to convert from String to &str?

Reply:
I think it's `.as_str()`